core/src/main/java/psiprobe/AbstractTomcatContainer.java

```diff
@@ -406,7 +406,7 @@ public void listContextJsps(Context context, Summary summary, boolean compile) {
   }
 
   @Override
-  public org.apache.juli.logging.Log getLogger(Context context) {
+  public Object getLogger(Context context) {
```

Comment:
Main objective is to make sure we use our own loggers and don't mistakening use tomcats.  In this case, not once is this method ever used other than as Object.  However, I see zero point for this in the interface or the concrete class.  It's 3 use cases simply call through the wrapper very pointlessly passing 'context' which then just reads context.getLogger().  The wrapper calls provide no additional processing and in my opinion are not needed.  Instead we should delete the interface method, concrete method in abstract class, and then just call it directly thus not wasting time calling through objects.

After deleting those, in LogResolverBean, the 3 calls all will be the same.

Previous
```
Object contextLogger = getContainerWrapper().getTomcatContainer().getLogger(ctx);
```
New would still just receive object back because it can be considered different in a couple of the calls plus setting wants Object not actual log type.
```
Object contextLogger = ctx.getLogger();
```
Please take a quick look over this to see if I'm missing anything in this actual use case.  I've got this switched locally as my thinking suggests above.  I'll be running some tests shortly.